package.json

```diff
@@ -35,6 +35,7 @@
     "license-checker-webpack-plugin": "0.2.1",
     "mermaid": "10.1.0",
     "mini-css-extract-plugin": "2.7.5",
+    "minimatch": "9.0.0",
```

Comment:
Good call on this. I was about to recommend `picomatch` but then I was reminded that it has node dependencies while `minimatch` seems to be free of those, so suitable for a browser.